pytest_asyncio/plugin.py

```diff
@@ -152,11 +154,39 @@ def pytest_pyfunc_call(pyfuncitem):
             return True
 
 
+def wrap_in_sync(func):
+    """Return a sync wrapper around an async function."""
+
+    @functools.wraps(func)
+    def inner(**kwargs):
+        loop = asyncio.get_event_loop_policy().new_event_loop()
```

Comment:
Why not bare `asyncio.new_event_loop()` here?

Reply:
`pytest-asyncio` works slightly different with event loops:
1. It installs the loop as default
2. Restores previously used default loop after the test finishing

Does your change follow this behavior?

Reply:
> Why not bare `asyncio.new_event_loop()` here?

There are so many ways to get an event loop that I just copied the `event_loop` fixture; I assume there's a reason it was chosen there.

> Does your change [set and restore the default loop]?

Not yet!  One moment...